packages/icons/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "@livechat/design-system-icons",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.ts",
+  "scripts": {
+    "build": "svgr --typescript -d ./dist ./src ",
+    "dev": "npm run build"
+  },
+  "author": "",
+  "files": [
+    "src"
```

Comment:
Are you sure 🤔 ? What do we want to expose? `src` or `dist`? :)

Reply:
my bad